distribution/src/main/packaging/systemd/elasticsearch@.service

```diff
@@ -0,0 +1,60 @@
+[Unit]
+Description=Elasticsearch Instance %i
+Documentation=http://www.elastic.co
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+RuntimeDirectory=elasticsearch
+Environment=ES_HOME=/usr/share/elasticsearch
+Environment=ES_PATH_CONF=${path.conf}/%i
+Environment=PID_DIR=/var/run/elasticsearch
+EnvironmentFile=-${path.env}-%i
+
+WorkingDirectory=/usr/share/elasticsearch
+
+User=elasticsearch
+Group=elasticsearch
+
+ExecStart=/usr/share/elasticsearch/bin/elasticsearch -p ${PID_DIR}/elasticsearch-%i.pid --quiet
+
+# StandardOutput is configured to redirect to journalctl since
+# some error messages may be logged in standard output before
+# elasticsearch logging system is initialized. Elasticsearch
+# stores its logs in /var/log/elasticsearch and does not use
+# journalctl by default. If you also want to enable journalctl
+# logging, you can simply remove the "quiet" option from ExecStart.
+StandardOutput=journal
+StandardError=inherit
+
+# Specifies the maximum file descriptor number that can be opened by this process
+LimitNOFILE=65536
```

Comment:
Agreed; I tried to come up with a better way with systemd natively but I couldn't come up with a good solution. For reference, here's some feedback from #systemd on freenode:

```
    leothrix | Am I mistaken, or am I remembering correctly that there's some sort of
             | `.import` option in unit files?
     zdzichu | leothrix: .include ?
     zdzichu | it's deprecated
    leothrix | zdzichu: Ah, that was the one I'd seen before. So it's being phased out?
     zdzichu | yep, .d/ snippets are better
     zdzichu | more clean and visible
    leothrix | Is there any way to define a new unit that includes the same options as
             | another unit instead of just overriding an existing unit?
boucman_work | leothrix: kinda, but that would be a bit ugly
     zdzichu | ha, that's one use case where .include would be helpful ;)
     zdzichu | right now, use 'cp'
boucman_work | your second unit would be an empty .service, then you create a .service.d/
             | directory and in there you put a symlink (renamed .conf) to your first unit
boucman_work | as I said : ugly
boucman_work | or put all the common parts in a .conf file, and symlink in both .d files (a
             | bit cleaner, but not exaclty what you asked)
```

So there's a deprecated way of doing so, and a hack to get the same effect, but nothing that looks ideal.